databricks/koalas/plot.py

```diff
@@ -427,7 +430,7 @@ def _compute_stats(data, colname, whis, precision):
         pdf = data._kdf._internal.resolved_copy.spark_frame.agg(
             *[
                 F.expr(
-                    "approx_percentile({}, {}, {})".format(colname, q, int(1.0 / precision))
+                    "approx_percentile(`{}`, {}, {})".format(colname, q, int(1.0 / precision))
```

Comment:
Good catch.

Also we might need to surround the `colname` in `_outliers`, `_calc_whiskers`, and `_get_fliers`.

- `F.mean(colname).alias("{}_mean".format(colname))`
- `expression = F.col(colname).between(lfence, ufence)`
- `outliers.filter("not __{}_outlier".format(colname))`
- `.agg(F.min(colname).alias("min"), F.max(colname).alias("max"))`
- `fliers_df = outliers.filter("__{}_outlier".format(colname))`